src/main/java/com/googleinterns/zoomtube/servlets/LectureServlet.java

```diff
@@ -71,17 +78,41 @@ public void init() throws ServletException {
   // TODO: Check if URL is valid.
   public void doPost(HttpServletRequest request, HttpServletResponse response)
       throws IOException, ServletException {
-    Optional<String> optionalVideoUrl = getParameter(request, LINK_INPUT);
-    Optional<Entity> existingEntity = checkUrlInDatabase(optionalVideoUrl);
+    if (request.getParameter(PARAM_NAME) == null) {
+      response.sendError(HttpServletResponse.SC_BAD_REQUEST, ERROR_MISSING_NAME);
+      return;
+    }
```

Comment:
Please do the same here as I mentioned in your other PR.

Reply:
Done